app/src/features/pricing/components/PremiumFeature/index.tsx

```diff
@@ -79,8 +77,11 @@ export const PremiumFeature: React.FC<PremiumFeatureProps> = ({
           },
         })
       );
+      return;
     }
-  }, [dispatch, hasCrossedDeadline, onContinue, source]);
+    redirectToUrl(LINKS.ACCELERATOR_PROGRAM_FORM_LINK, true);
+    trackUpgradeOptionClicked("upgrade_for_6_months");
+  }, [dispatch, source, isIncentivizationEnabled, onUpgradeForFreeClickCallback]);
```

Comment:
unrelated to these changes, but `source` can be removed from dependency array here